rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -198,14 +227,18 @@ type CollateralTaxonomyValue: <"Specifies the collateral taxonomy value, either
     condition:
         one-of
 
-type Security extends ProductBase: <"Identifies a security by referencing a product identifier and by specifying the sector.">
+type Security extends InstrumentBase: <"Identifies a security by referencing a product identifier and by specifying the sector.">
     [docReference ICMA GMRA namingConvention "Purchased Security"
 		provision "As defined in GMRA paragraph 2(oo) The Purchased Securities are the Securities sold or to be sold and any New Purchased Securities transferred by Seller to Buyer under paragraph 8 (Substitution)."]
     securityType SecurityTypeEnum (1..1) <"Identifies the type of security using an enumerated list.">
     debtType DebtType (0..1) <"Identifies the type of debt and selected debt economics.">
     equityType EquityTypeEnum (0..1) <"Identifies the type of equity.">
     fundType FundProductTypeEnum (0..1) <"Identifies the type of fund.">
+
+    // temporary items
```

Comment:
Can you confirm when these temporary items will be addressed please? Is it to be done in the Phase 3 PR?